projects/storefrontlib/src/lib/my-account/order/order-details/order-detail-items/order-detail-items.component.ts

```diff
@@ -27,4 +47,15 @@ export class OrderDetailItemsComponent implements OnInit {
 
     return products;
   }
+
+  getConsignmentTracking(consignment: Consignment) {
```

Comment:
Should it be called something like `open` or `show` instead of `get`, to better describe its behavior?